src/Plugin.hpp

```diff
@@ -27,10 +29,10 @@ class Plugin : public cs::core::PluginBase {
       std::string mTexture;
 
       /// The distance from the planets center to where the rings start in meter.
-      double mInnerRadius;
+      float mInnerRadius{};
 
       /// The distance from the planets center to where the rings end in meter.
-      double mOuterRadius;
+      float mOuterRadius{};
```

Comment:
Why use float here? Shouldn't we use doubles for as long as possible? 

Doubles have better precision especially on the scales we are working on, and on x86 architecture they have the same or even better performance than floats. The only downside is memory consumption, which can be ignored in simple cases like this.

Reply:
In general, I agree with you. But here they are not used for any calculations, they are directly passed to the shader. So we wouldn't get any benefit, on the contrary - we would have to static_cast them every frame.